bazel/foreign_cc/BUILD

```diff
@@ -0,0 +1 @@
+licenses(["notice"])  # Apache 2
```

Comment:
Did you intend to commit this?

Reply:
a `BUILD` file is required. i don't think the `licenses` line is necessary, but i don't think it's harmful either

Reply:
Ah I see, I didn't realize there was no `BUILD` file in `bazel/foreign_cc` before this. So the reason that wasn't an issue before is that the patches weren't actually being consumed?

Reply:
added [this comment](https://github.com/solo-io/envoy-gloo/pull/236/commits/e17f5cfe1391c1f14549ebe639eb0ce16b2cc248) to explain why the file is needed

Reply:
Looks good, resolving this thread